src/App.js

```diff
@@ -1,28 +1,18 @@
-import React, { useState, useEffect } from 'react';
-import { getAllData } from './util/index';
+import React from "react";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
 import LandingPage from "./components/LandingPage";
-
-const URL = 'http://localhost:8000/api/v1/';
+import Register from "./components/Register";
 
 function App() {
-  const [message, setMessage] = useState('');
-
-  useEffect(() => {
-    (async () => {
-      const myData = await getAllData(URL);
-      setMessage(myData.data);
-    })();
-
-    return () => {
-      console.log('unmounting');
-    };
-  }, []);
-
   return (
-    <>
-      {/*<h1>{message}</h1>*/}
-      <LandingPage />
-    </>
+    <BrowserRouter basename="/">
```

Comment:
I haven't seen this basename prop before, but chatGPT says it's for "multi-page apps or apps that are not hosted at the root of the domain."

I think for our case we may not need this since neither of those apply, though I don't think it's hurting anything!